pig/src/main/java/org/jboss/pnc/bacon/pig/impl/utils/BuildFinderUtils.java

```diff
@@ -135,7 +135,7 @@ public static BuildConfig getKojiBuildFinderConfig() {
             log.error("Koji client error: {}", e.getMessage(), e);
             return Collections.emptyList();
         } finally {
-            pool.shutdown();
+            Utils.shutdownAndAwaitTermination(pool);
```

Comment:
This does more than just `shutdown` in accordance with the documentation at https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/ExecutorService.html.